apiFunctions.php

```diff
@@ -325,8 +325,12 @@ function expand_by_doi($template, $force = FALSE) {
   // there will be few instances where it could not in principle be profitable to 
   // run this function, so we don't check this first.
   
-  $doi = $template->get_without_comments_and_placeholders('doi');
-  if (!$template->verify_doi()) return FALSE;
+  if (!$template->verify_doi()) { // Try non-crossref DOI resolver
```

Comment:
Presumably we also need to modify `verify_doi` such that it recognizes valid non-CrossRef DOIs?  If so, we'll need a different test in this `if`.

Reply:
The real goal of verify doi is to to see if the crossref rich api is available or if we have to use doi.org.

Reply:
Maybe the name of the function needs changed